test/unit/karma.conf.js

```diff
@@ -44,7 +44,11 @@ module.exports = function (config) {
             { pattern: 'test/unit/testUtil.js', nocache: true },
             { pattern: 'test/unit/mocks.js', nocache: true },
             { pattern: 'test/unit/test-main.js', nocache: true },
-            { pattern: 'test/unit/spec/**/*.js', included: false, nocache: true },
+            {
+                pattern: 'test/unit/spec/appWidgets/view/newObjectViewSpec.js',
```

Comment:
what's this change for?

Reply:
Oh, I think my last commit upload (which fixed that) failed.